### ISSUE TITLE
Solutions missing

### ISSUE THREAD

**Issue by JDBetteridge** [arc-bath/now-code#3]
These probably won't be made available for the lesson, but there should probably a placeholder page for the time being (easy enough with a redirect)

**Comment by james-grant1** [comment]
Yes, they are, thanks for pointing this out, I have no intention of providing them at this stage so will need to change everything to pens.  Ta for other comments/PR will look through.
